### PR TITLE
Add ca_trusted_fingerprint documentation to standalone agent docs

### LINKED ISSUE #1308: Update Elastic Agent and Fleet docs to include ES CA fingerprints
For Standalone agent and managed agent.

**Owner**
@michel-laterman

### PR DESCRIPTION
**Pull request by michel-laterman**
Add missing documentation for using the `elastic-agent` in stand-alone mode.

- Closes #1308

### CODE REVIEW

File: docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc [hunk 1]
```diff
@@ -90,6 +90,7 @@ sudo ./elastic-agent install  -f \
   --fleet-server-es=http://localhost:9200 \
   --fleet-server-service-token=AAEbAWVsYXN0aWMvZmxlaXQtc2VydmVzL3Rva2VuLTE2MeIzNTY1NTQ3Mji6dERXeE9XbW5RRTZqNlJMWEdIRzAtZw \
   --fleet-server-policy=27467ed1-1bfd-11ec-9b88-a7c3d83e2897
```
Comment (dedemorton) [inline docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc:92]:
```suggestion
  --fleet-server-policy=27467ed1-1bfd-11ec-9b88-a7c3d83e2897 \
```

File: docs/en/ingest-management/elastic-agent/configuration/authentication/ssl-shared-settings.asciidoc [hunk 2]
```diff
@@ -333,6 +333,25 @@ production environments is strongly discouraged.
...
+// tag::ssl.ca_trusted_fingerprint
+|
+[id="{type}-ssl.ca_trusted_fingerprint"]
+`ssl.ca_trusted`
+`_fingerprint`
+
+| (string) A HEX encoded SHA-256 of a CA certificate. If this certificate is
+present in the chain during the handshake, it will be added to the
+`certificate_authorities` list and the handshake will continue
+normaly.
```
Comment (dedemorton) [inline docs/en/ingest-management/elastic-agent/configuration/authentication/ssl-shared-settings.asciidoc:345]:
```suggestion
normally.
```

**Review by dedemorton — COMMENTED** [review]
Could of minor nits. Otherwise looks good.

**Review by belimawr — COMMENTED** [review]
It looks great! I'm just waiting for you to address the comments from @dedemorton

**Comment by dedemorton** [comment]
One other thing that I just noticed... because of how tables work in our docs, the long token creates a weird situation where users need to scroll to the right to see descriptive text. Maybe you should shorten the fingerprint (maybe use ... or something to indicate missing characters). I wish we didn't have to hack the content to make the layout work, but this seems like such a user experience.

image

**Review by dedemorton — APPROVED** [review]
LGTM. Thanks for putting this PR up!!

**Pull request merged**